src/main/resources/assets/someassemblyrequired/lang/de_de.json

```diff
@@ -0,0 +1,110 @@
+{
+  "itemGroup.someassemblyrequired": "Some Assembly Required",
+
+  "recipecategory.someassemblyrequired.cutting": "Schneiden",
+  "recipecategory.someassemblyrequired.toasting": "Toasten",
+
+  "message.someassemblyrequired.top_bread": "Obere Brotscheibe fehlt!",
+  "message.someassemblyrequired.bottom_bread": "Untere Brotscheibe fehlt!",
+  "message.someassemblyrequired.full_sandwich": "Sie können nichts weiteres hinzufügen!",
+
+  "block.someassemblyrequired.acacia_sandwich_assembly_table": "Akazien-Sandwich-Montagetisch",
+  "block.someassemblyrequired.birch_sandwich_assembly_table": "Birken-Sandwich-Montagetisch",
+  "block.someassemblyrequired.crimson_sandwich_assembly_table": "Purpurroter Sandwich-Montagetisch",
+  "block.someassemblyrequired.dark_oak_sandwich_assembly_table": "Dunkeleichen-Sandwich-Montagetisch",
+  "block.someassemblyrequired.jungle_sandwich_assembly_table": "Dschungel-Sandwich-Montagetisch",
+  "block.someassemblyrequired.oak_sandwich_assembly_table": "Eichen-Sandwich-Montagetisch",
+  "block.someassemblyrequired.spruce_sandwich_assembly_table": "Fichten-Sandwich-Montagetisch",
+  "block.someassemblyrequired.warped_sandwich_assembly_table": "Verzogener-Sandwich-Montagetisch",
+  "block.someassemblyrequired.acacia_cutting_board": "Akazien Schneidbrett",

```

Comment:
Isn't it "Schneidebrett"?

Reply:
thanks for all the suggestions, was pretty tired when doing this last night 😄 . Agreed it is Schneidebrett.